addons/project/models/project_update.py

```diff
@@ -74,14 +77,23 @@ def _compute_name_cropped(self):
         for u in self:
             u.name_cropped = (u.name[:57] + '...') if len(u.name) > 60 else u.name
 
+    def _compute_closed_task_percentage(self):
+        for u in self:
+            u.closed_task_percentage = u.task_count and round(u.closed_task_count * 100 / u.task_count)
```

Comment:
We can be more explicit. 🙂 
```suggestion
        for update in self:
            update.closed_task_percentage = update.task_count and round(update.closed_task_count * 100 / update.task_count)
```
or even more explicit
```suggestion
        for project_update in self:
            project_update.closed_task_percentage = project_update.task_count and round(project_update.closed_task_count * 100 / project_update.task_count)
```